api_factory/generator/templates/$namespace/$name_$version/$service/client.py.j2

```diff
@@ -51,6 +55,20 @@ class {{ service.name }}:
         else:
             Transport = get_transport_class(transport)
             self._transport = Transport(credentials=credentials)
+        {%- if service.has_lro %}
+
+        # Instantiate a client for the long-running operations
+        if hasattr(self._transport, 'grpc_channel'):
```

Comment:
The plan for this part of the code:
1. implement flattening
2. generate an operations client in `api_core` that uses transports
3. Replace this whole section with `self._operations_client = operations.Operations(self._transport)`

I think adding a channel-generating method to the base transport would end up being messier. The logic follows that if you have a `grpc_channel`, use it. Otherwise you still need one, so create it.

Reply:
Nvm discussed

Reply:
In the interest of having a record, the crux of the discussion was:

We decided that it was cleaner to have the LRO client be returned from the _transport_, rather than the client, for the following reasons:

  * Simply passing the string transport from the Client constructor to the OperationsClient constructor feels brittle.
  * It makes it clearer that a custom transport must implement a client for LRO (or use one of the ones we provide).